src/shadowbox/server/server_access_key.ts

```diff
@@ -182,7 +188,9 @@ export class ServerAccessKeyRepository implements AccessKeyRepository {
       encryptionMethod,
       password,
     };
-    const accessKey = new ServerAccessKey(id, '', metricsId, proxyParams);
+    const name = params?.name ?? this.NEW_USER_DEFAULT_NAME;
```

Comment:
This is more readable:

```suggestion
    const name = params?.name ?? '';
```